pandas/core/base.py

```diff
@@ -877,6 +877,10 @@ def _values(self):
         """ the internal implementation """
         return self.values
 
+    @property
+    def empty(self):
+        return not self.size
```

Comment:
you can easily fix the doc-strings by using the _shared_docs. ok with having one for Index and one in generic also.